### PR TITLE
Etn/pipeline improvements

### PR DESCRIPTION
**Pull request by ed-nykaza**
@pazaan, I refactored a few things given the other changes we recently made. Please review when you get a chance, but this review is not urgent.

### CODE REVIEW

File: tidals/clean.py [hunk 1]
```diff
@@ -63,3 +64,46 @@ def round_time(df, timeIntervalMinutes=5, timeField="time",
...
+        df.loc[df[fieldName].notnull(), fieldName] = \
+            df.loc[df[fieldName].notnull(), fieldName].str[0]
+
+    return df
+
+
+def flatten_json(df):
+
+#    # remove fields that we don't want to flatten
+#    df, holdData = tempRemoveFields(df)
```
Comment (pazaan) [inline tidals/clean.py]: Why are you commenting out this code? And if you do mean to get rid of it, why not just delete it? Same goes further down.

File: tidals/clean.py [hunk 2]
```diff
@@ -63,3 +64,46 @@ def round_time(df, timeIntervalMinutes=5, timeField="time",
...
+            jsonBlob = df[colHead][df[colHead].astype(str).str[0] == "{"]
+
+            # replace those values with nan
+            df.loc[jsonBlob.index, colHead] = np.nan
+
+            # turn jsonBlob to dataframe
+            newDataFrame = pd.concat([newDataFrame, pd.DataFrame(jsonBlob.tolist(),
+                                      index=jsonBlob.index).add_prefix(colHead + '.')], axis=1)
+
+#    newColHeadings = list(newDataFrame)
```
Comment (pazaan) [inline tidals/clean.py]: See above :)

File: tidals/clean.py (hunk 1 above)
Comment (ed-nykaza) [reply tidals/clean.py]: good catch...thanks...I have removed the commented code

**Comment by ed-nykaza** [comment]
@pazaan, this is ready for approval

**Pull request merged**